lib/Mojo/WebSocketProxy/Dispatcher.pm

```diff
@@ -52,7 +52,7 @@ sub open_connection {
 
     my $log            = $c->app->log;
     my $remote_address = $c->tx->remote_address // 'undef';
-    $log->debugf('accepting a websocket connection from ' . $remote_address);
+    $log->debug('accepting a websocket connection from ' . $remote_address);
     # Enable permessage-deflate
```

Comment:
interpolation is not recommended https://deriv-group.slack.com/archives/C0D442KHV/p1701738911448829
can we show log in case its defined only

Reply:
Thank you. Took out string interpolation